testdata/p4_16_samples_outputs/v1model-digest-containing-ser-enum.p4.p4info.txt

```diff
@@ -263,15 +263,15 @@ type_info {
       }
       members {
         name: "foo"
-        value: "28"
+        value: "\034"
       }
       members {
         name: "bar"
-        value: "17"
+        value: "\021"
       }
       members {
         name: "gah"
-        value: "42"
+        value: "*"
```

Comment:
Where is this "*" coming from?  Judging by the changes several lines above, shouldn't it be `"\052"`, i.e. escaped octal for the decimal value 42?

Reply:
Oh, I guess judging from the previous file's diffs, `"*"` has the ASCII value of decimal 42?  Kind of confusing the first time you see this.

Reply:
yes these values prioritize size over human-readability :)

I got confused when I reviewed my diff as well...